.circleci/config.yml

```diff
@@ -225,11 +230,6 @@ jobs:
         steps:
             - prepare-rtl:
                 project-key: "chipyard-boom"
-    prepare-rocketchip:
-        executor: main-env
-        steps:
-            - prepare-rtl:
-                project-key: "rocketchip"
```

Comment:
The "rocketchip" set of tests did the exact same thing as the "chipyard-rocket" set of tests, so I removed these.